docs/modules/reference/pages/billing/signup/overview.adoc

```diff
@@ -0,0 +1,46 @@
+= Managing Subscriptions in Payara Cloud
+
+Payara Cloud uses subscriptions as the primary billing unit.
+All application usage across various namespaces gets consolidated into a single invoice, simplifying the billing process.
+
+A single Billing Manager can oversee multiple subscriptions.
+These subscriptions can be associated with different companies and utilize various payment methods.
+
+== Starting New Subscription
+
+=== Free Trial
+
+For new users, the best way to begin is with our free trial.
+A detailed guide to get you started is available here: xref:docs:ROOT:getting-started/cloud-trial/Signup Payara Cloud.adoc[].
+
+If you're an existing Payara Cloud user, you can easily create additional subscriptions.
+You have the option to either link:additional.adoc[reusing existing invoicing data] or link:additional.adoc[provide new invoicing data] for these new subscriptions.
+
+== Cancelling Subscription
+
+A subscription can be cancelled at any time by following procedure listed in xref:docs:ROOT:getting-started/cloud-trial/Cancel Payara Cloud.adoc[].
+
```

Comment:
I'm not sure why the cancel doc isn't created as a link in the rendering.

Reply:
Because the title had a stray `:` at beginning of line